interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -157,6 +187,7 @@ public void emptyUnary() throws Exception {
 
   @Test(timeout = 10000)
   public void largeUnary() throws Exception {
+    context.attach();
```

Comment:
I attach all to ensure that the assertions done by TestServiceImpl apply to all operation types.

This is not done for InProcessTransport, it is done for tests where TestServiceImpl is running in the same process as the test.

Note that TestServiceImpl.CLIENT_ONLY_CONTEXT is used to ensure the InProcessTransport isn't carrying contexts from the client application thread to the server application thread.